Makefile

```diff
@@ -70,6 +70,9 @@ venv:
 	if [ ! -z "${VIRTUAL_ENV}" ]; then \
 	  echo "Found an activated Python virtual environment, exiting" && exit 1; \
 	fi
+	if [ -d .venv/ ]; then \
+	  echo "Found an inactive Python virtual environment, please activate" && exit 1; \
```

Comment:
It might be good to ask to "Remove it otherwise."?

Reply:
You mean as part of the `echo`?

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/345/commits/c183f13566e3cbd3de2e58b9f859e1ebc42c15f8.